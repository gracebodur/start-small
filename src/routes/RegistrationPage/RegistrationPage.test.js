import React from "react";
import toJson from "enzyme-to-json";
import RegistrationPage from "./RegistrationPage";

describe(`RegistrationPage component`, () => {
  const props = {
    id: "a",
    name: "test-class-name"
  };

  it("renders a RegistrationPage by default", () => {
    const wrapper = <RegistrationPage />;
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it("renders the RegistrationPage given props", () => {
    const wrapper = <RegistrationPage {...props} />;
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
