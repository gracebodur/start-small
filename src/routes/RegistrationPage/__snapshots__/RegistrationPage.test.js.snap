// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegistrationPage component renders a RegistrationPage by default 1`] = `undefined`;

exports[`RegistrationPage component renders the RegistrationPage given props 1`] = `undefined`;
